### PR TITLE
Update batch

### PR DESCRIPTION
**Pull request by knockknockyoo**
1. update batch error codes
2. searching batch by its id
3. update batch user-control
4. relocated organization service after permission - more efficient logic

### CODE REVIEW

**Review by AustinSMueller — APPROVED** [review]

File: services/submission.js
Comment (vshand11) [inline services/submission.js:1]: VerifySession or VerifyToken is missing in updateBatch function.

**Review by vshand11 — CHANGES_REQUESTED** [review]

**Comment by vshand11** [comment]
Approved.

**Review by vshand11 — APPROVED** [review]
approved

**Comment by vshand11** [comment]
Passed my testing with session and token.

**Pull request merged**